src/gui/input.cpp

```diff
@@ -195,6 +195,13 @@ QString convertKey(const QKeyEvent& ev) noexcept
 		return ToKeyString(GetModifierPrefix(mod), "Bslash");
 	}
 
+	// Issue#833: Handle MacOS Keyboard
+#ifdef __APPLE__
+	if (mod & Qt::AltModifier ) {
```

Comment:
I think this is too broad. Does this pass Unit Tests?

Consider the case where a user wants to map `<A-X>` on a US layout.

Make sure to run the unit tests:
```
cd build
cmake --build .
ctest -R tst_input*
```